consensus/crdt/consensus.go

```diff
@@ -330,9 +336,19 @@ func (css *Consensus) Shutdown(ctx context.Context) error {
 		logger.Debug("already shutdown")
 		return nil
 	}
+	css.shutdown = true
 
 	logger.Info("stopping Consensus component")
 
+	// Cancel
+	css.batchingCancel()
+	if css.config.batchingEnabled() {
```

Comment:
this does not really work, it is still racy, we would need to wait until the batch worker has existed to be sure nothing else will be committed.